android/java/org/chromium/chrome/browser/brave_news/BraveNewsUtils.java

```diff
@@ -0,0 +1,128 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
this file needs `git cl format  android/java/org/chromium/chrome/browser/brave_news/BraveNewsUtils.java`